packages/executor/src/modules/eth.ts

```diff
@@ -459,7 +459,7 @@ export class Eth {
         ov.perUserOp +
         ov.perUserOpWord * lengthInWord
     );
-    return ret;
+    return ret + this.config.pvgMarkup;
```

Comment:
Shouldn't we add validation here to ensure the values are positive?

Reply:
`ret` is always positive and `pvgMarkup` is negative only when deliberately set to a negative value by a relayer, so I think validation is excessive